src/v2/Apps/Fair/Components/FairFollowedArtists.tsx

```diff
@@ -17,6 +24,45 @@ export const FairFollowedArtists: React.FC<FairFollowedArtistsProps> = ({
   fair,
   ...rest
 }) => {
+  const tracking = useTracking()
+
+  const {
+    contextPageOwnerId,
+    contextPageOwnerSlug,
+    contextPageOwnerType,
+  } = useAnalyticsContext()
+
+  const tappedViewTrackingData: ClickedArtworkGroup = {
+    context_module: ContextModule.worksByArtistsYouFollowRail,
+    context_page_owner_type: contextPageOwnerType,
+    context_page_owner_id: contextPageOwnerId,
+    context_page_owner_slug: contextPageOwnerSlug,
+    destination_page_owner_type: OwnerType.fair,
+    destination_page_owner_id: fair.id,
```

Comment:
`internalID` not `id`! (`id` is relay-specific)

Reply:
Good catch thank you! — addressed